### ISSUE TITLE
Kirby and the forgotten land Random softlocks

### ISSUE THREAD

**Issue by walikor** [yuzu-emu/yuzu#11231]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1518

### Description of Issue

the game randomly goes into softlocks, there is no precise time, it seems to happen only with Vulkan API

### Expected Behavior

the game shouldn't softlocks

### Reproduction Steps

play the game

### Log File

https://www.mediafire.com/file/oob00ciybmdl92r/yuzu_log.txt/file

https://ibb.co/JCmStdH image

### System Configuration

CPU: HexaCore Intel Core i5-8500, 3.00 GHz
GPU/Driver: NVIDIA GeForce GTX 1080 (8 GB) driver version 536.67
RAM: 2x Corsair Vengeance LPX CMK8GX4M1A2400C14 8 GB ( 16 GB)
OS: Microsoft Windows 10 Pro 22H2

**Comment by EgoBizarro** [comment]
Yeah, it's the good old Pascal card + yuzu versions since YFC1.5 + Vulkan = softlock/crash
Seems like nothing can be done about it (that's why I hope they don't get rid of OpenGL any time soon). Unfortunately, in this game, OpenGL has strange performance issues, so I can't really play the game on yuzu.

**Comment by Squall-Leonhart** [comment]
> Yeah, it's the good old Pascal card + yuzu versions since YFC1.5 + Vulkan = softlock/crash

no its not.

**Comment by EgoBizarro** [comment]
Oh, no device loss in the log? Huh, well, whatever that was, device loss would have come later anyway; I can never play half an hour without it happening sooner or later.

**Comment by walikor** [comment]
> Oh, no device loss in the log? Huh, well, whatever that was, device loss would have come later anyway; I can never play half an hour without it happening sooner or later.

https://www.mediafire.com/file/se06x4g17tm0i37/yuzu_log.txt/file

this is another log file, i was playing vulkan and the game crashed but the music was still playing.

**Comment by Squall-Leonhart** [comment]
[   9.646374] Config <Info> common\settings.cpp:operator ():102: M- Renderer.async_presentation: true
[   9.646374] Config <Info> common\settings.cpp:operator ():102: M- Renderer.force_max_clock: true

set these back to default.

**Comment by walikor** [comment]
> [ 9.646374] Config common\settings.cpp:operator ():102: M- Renderer.async_presentation: true [ 9.646374] Config common\settings.cpp:operator ():102: M- Renderer.force_max_clock: true
> 
> set these back to default.


it seems that by setting these two options to default as you told me the game seems to work, I did 5 levels without ever having softlocks, in case it recurs I'll let you know, but I think it's solved.

**Comment by walikor** [comment]
> [ 9.646374] Config common\settings.cpp:operator ():102: M- Renderer.async_presentation: true [ 9.646374] Config common\settings.cpp:operator ():102: M- Renderer.force_max_clock: true
> 
> set these back to default.

forget it.... same problem

**Comment by Squall-Leonhart** [comment]
Redump the game, i'm sorry but i don't have this issue

**Comment by iyotee** [comment]
The game hardly softcrash in the 'Windy Freezy Sees' level no matter what hardware is used ( also happened to a friend ) even if you change the dumped game to another one.

**Comment by shehzman** [comment]
Same issue for me with EA 3891

**Comment by walikor** [comment]
> Same issue for me with EA 3891. Same issues in all the levels mentioned above and Scale the Cement Submit. Turning the 60FPS patches off seems to fix it

if you use the 60 fps patch with GPU high and CPU unsafe,do you have same problem?

**Comment by shehzman** [comment]
yup

**Comment by Squall-Leonhart** [comment]
> Oh, no device loss in the log? Huh, well, whatever that was, device loss would have come later anyway; I can never play half an hour without it happening sooner or later.

> forget it.... same problem, if it helps, using opengl doesn't seem to have the problem

Untick "Barrier Feedback loops" under Graphics>Advanced
